### PR TITLE
Update grid doc for consistent use of container wrapper.

### PR DESCRIPTION
**Pull request by florianlacreuse**
In the [grid section](https://getbootstrap.com/docs/4.1/layout/grid/) of the documentation, some examples use `.container` parent class and some don't. This may be confusing for new users. Furthermore, not all examples are aligned in the same way (since the missing paddings).

### CODE REVIEW

File: site/docs/4.1/layout/grid.md [hunk 1]
```diff
@@ -302,38 +308,42 @@ Don't want your columns to simply stack in some grid tiers? Use a combination of
...
+Gutters can be responsively adjusted by breakpoint-specific padding and negative margin utility classes. To change the gutters in a given row, pair a negative margin utility on the `.row` and matching padding utilities on the `.col`s. The `.container` or `.container-fluid` parent may need to be adjusted too to avoid unwanted overflow, using again matching padding utility.
 
-Here's an example of customizing the Bootstrap grid at the large (`lg`) breakpoint and above. We've increased the `.col` padding with `.px-lg-5` and then counteracted that with `.mx-lg-n5` on the parent `.row`.
+Here's an example of customizing the Bootstrap grid at the large (`lg`) breakpoint and above. We've increased the `.col` padding with `.px-lg-5`, counteracted that with `.mx-lg-n5` on the parent `.row` and then adjusted the `.container` wrapper with `.px-lg-5`.
 
 {% capture example %}
-<div class="row mx-lg-n5">
-  <div class="col py-3 px-lg-5 border bg-light">Custom column padding</div>
-  <div class="col py-3 px-lg-5 border bg-light">Custom column padding</div>
+<div class="container px-lg-5">
```
Comment (florianlacreuse) [inline site/docs/4.1/layout/grid.md]: I'm not sure about this one, may be we don't want to talk about `.container` here?

Comment (MartijnCuppens) [reply site/docs/4.1/layout/grid.md]: On gigantic screens or when you're zoomed out a lot you'll see that this class will align this example with the rest, you can keep it here.

**Review by MartijnCuppens — APPROVED** [review]
I like this, this keeps the examples aligned. Nice job @florianlacreuse!

**Comment by florianlacreuse** [comment]
@MartijnCuppens Thanks! I hope this will be merged for the next v4.2 coming soon.

**Review by mdo — APPROVED** [review]

**Pull request merged**